backend/cesium_api/app.py

```diff
@@ -27,9 +29,12 @@ def __init__(
         self.host = host
         self.port = port
         self.data_dir = data_dir
+        self.tiles_dir = tiles_dir or DEFAULT_TILES_DIR
```

Comment:
```suggestion
```

See related comment below about `DEFAULT_DATA_DIR` vs `DEFAULT_TILES_DIR`.

But make sure to remove this here.

On a side note, the use of `or` here is clever but not necessary, since the constructor parameter already has a default set via `tiles_dir: str = DEFAULT_TILES_DIR`